src/main/java/com/gregtechceu/gtceu/integration/ae2/gui/widget/AmountSetWidget.java

```diff
@@ -43,28 +36,25 @@ public void setSlotIndex(int slotIndex) {
         this.index = slotIndex;
     }
 
-    public String getAmountStr() {
-        if (this.index < 0) {
-            return "0";
+    public long getAmount() {
+        if (index < 0) {
```

Comment:
very tiny nitpick, either use this.index both here and in setAmount, or just index in both <3